src/main/java/uk/gov/ons/ctp/response/sample/SampleSvcApplication.java

```diff
@@ -58,6 +59,13 @@ public static void main(final String[] args) {
     SpringApplication.run(SampleSvcApplication.class, args);
```

Comment:
Need to remove `LoggingConfigs.setCurrent(LoggingConfigs.getCurrent().useJson());`